packages/@contentlayer/utils/src/node/fs.ts

```diff
@@ -95,6 +95,30 @@ export const mkdirp = (dirPath: string): T.Effect<OT.HasTracer, MkdirError, void
     ),
   )
 
+export function rm(path: string, params: { force: true }): T.Effect<OT.HasTracer, RmError, void>
+export function rm(
+  path: string,
+  params?: { force: false },
+): T.Effect<OT.HasTracer, RmError | FileOrDirNotFoundError, void>
+
+export function rm(
+  path: string,
+  params = { force: false },
+): T.Effect<OT.HasTracer, RmError | FileOrDirNotFoundError, void> {
+  return OT.withSpan('rm', { attributes: { path } })(
+    T.tryCatchPromise(
+      () => fs.rm(path, { recursive: true, force: params.force }),
```

Comment:
I suggest we also make `recursive` configurable and default `true`. WDYT?

Reply:
Definitely makes sense! Added in https://github.com/contentlayerdev/contentlayer/pull/19/commits/74156f82931da2dd7ac9bc41ff42789df4632397